detect/fixeddetector.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from .detector import Detector
+from dataset import Table
+
+class FixedDetector(Detector):
+    def __init__(self, name, src, fpath=None, 
```

Comment:
We need more documentation about the class and the `__init__` parameters

Reply:
Added comments.